.gitignore

```diff
@@ -21,3 +21,4 @@ erl_crash.dump
 
 # Ignore the IEx bindings file
 .iex.exs
+.elixir_ls
```

Comment:
new line ?

Reply:
@kartikjagdale it was my local file wrongly committed.